samples/dapr/AppHost/Program.cs

```diff
@@ -1,9 +1,16 @@
+using Aspire.Hosting.Dapr;
+
 var builder = DistributedApplication.CreateBuilder(args);
 
 builder.AddDapr();
 
+string resourcesRelativePath = @"../Resources";
+string stateStoreRelativePath = Path.Combine(resourcesRelativePath, "statestore.yaml");
```

Comment:
This will be problematic depending on where you run from. I wonder if there's anything we can do here to make this stable. Maybe an env variable in launch settings?

Reply:
The intent was that component file paths be relative to the referenced project file (directory), which would be similar to how paths are expressed when using Dapr run files.  That still could problematic if the same component is referenced by multiple projects but where those projects are not at the same relative location in the solution.  A environment/configuration value might be ok...I'll think about it a bit.

Reply:
We made a fix in my latest PR. You can get the apphost project directory from the builder so it's stable no matter where you run from. This can be a follow up PR.